static/js/compose_error.js

```diff
@@ -0,0 +1,32 @@
+import $ from "jquery";
+
+import * as common from "./common";
+
+export function compose_error(error_html, bad_input) {
+    $("#compose-send-status")
+        .removeClass(common.status_classes)
+        .addClass("alert-error")
+        .stop(true)
+        .fadeTo(0, 1);
+    $("#compose-error-msg").html(error_html);
+    $("#compose-send-button").prop("disabled", false);
+    $("#sending-indicator").hide();
+    if (bad_input !== undefined) {
+        bad_input.trigger("focus").trigger("select");
+    }
+}
```

Comment:
I think we also need a `compose_error.hide()` method, for clearing the error when we're done with it?

And  any click handlers for e.g. closing them via the `x` should go in this file too.